.github/workflows/release-embedding-sdk.yml

```diff
@@ -59,11 +59,16 @@ jobs:
           result-encoding: string
           script: |
             const currentVersion = "${{ steps.current-sdk-version.outputs.sdk_current_version }}";
-            const versionParts = currentVersion.split(".");
+            const [currentVersionWithoutSuffix, suffix] = currentVersion.split("-");
+            const versionParts = currentVersionWithoutSuffix.split(".");
             versionParts[versionParts.length - 1] = parseInt(versionParts.at(-1)) + 1;
             const newVersion = versionParts.join(".");
 
-            return newVersion
+            if (!suffix) {
+              return newVersion;
+            }
+
+            return `${newVersion}-${suffix}`
```

Comment:
This new logic is [tested here](https://github.com/metabase/metabase/pull/49883#issue-2652462000). You can see the result in the `Demo` section.